### PR TITLE
Drop Ubuntu Xenial support

### PR DESCRIPTION
**Pull request by paddatrapper**
Xenial installation is currently broken anyway, due to Python dropping 3.5 support. We depend on pip packages for many of the dependencies we use (and run `pip install pip --upgrade` during install). With 3.5 not being supported, many of the packages we need are no longer available.

This does make a direct, in-place upgrade from Airtime impossible, but this is still supported through importing the Airtime DB and library after running the install script. This still needs testing though. The user would need to manually update and fix the configuration file. I have added documentation to this effect

Fixes: #1226
Fixes: #71 
Fixes: #623

### CODE REVIEW

**Comment by zklosko** [comment]
```
ubuntu@lt-old:~/libretime$ sudo ./install -fiap

.____    ._____.               ___________.__                
|    |   |__\_ |_________   ___\__    ___/|__| _____   ____  
|    |   |  || __ \_  __ \_/ __ \|    |   |  |/     \_/ __ \ 
|    |___|  || \_\ \  | \/\  ___/|    |   |  |  Y Y  \  ___/ 
|_______ \__||___  /__|    \___  >____|   |__|__|_|  /\___  >
        \/       \/            \/                  \/     \/

Detecting distribution and release ...
Detected distribution id: ubuntu
Detected distribution release id: 16.04
Detected distribution description: Ubuntu 16.04.7 LTS
ERROR: Ubuntu Xenial is archived and does not receive any security or other updates since 2021-04-01.
The LibreTime installer dropped support for installing LibreTime on Xenial in 3.0.0-alpha.10.
ubuntu@lt-old:~/libretime$ 
```

I can confirm your code works.

**Review by zklosko — APPROVED** [review]
Works great on Multipass!

**Comment by zklosko** [comment]
Please make sure you update our system requirements on the Install page in the docs.

**Comment by paddatrapper** [comment]
@Robbt any thoughts on this?

**Comment by jooola** [comment]
This changes mostly requires some background with Airtime, but I think it's quiet good. I think the backup and clean reinstall is the best way to migrate from Airtime. The only small thing is cosmetic: I would rather try to write a step by step guide like the "Upgrade form Airtime" section than have a big text block.

But yeah, my 2cts.

**Comment by paddatrapper** [comment]
What section are you referring to? The upgrading from Airtime section is step by step in this PR

File: docs/_docs/upgrading.md [hunk 1]
```diff
@@ -27,20 +26,46 @@ intended compatibility requirements as per semver.
...
+of how to perform these back ups.
+
+The LibreTime [installation script](/install) will detect an existing LibreTime
+deployment and back up any configuration files that it finds. We recommend
+taking your own manual backups of the configuration yourself nevertheless. The
+install script also tries to restart the needed services during an upgrade. In
+any case you should monitor if this happened and also take a quick look at the
+logs files to be sure everything is still fine. Now might be the time to reboot
+the system or virtual machine LibreTime is running on since regular reboots are
+part of a healthy system anyway.
```
Comment (jooola) [inline docs/_docs/upgrading.md:44]: This block and the block beneath it.

**Comment by Robbt** [comment]
Yeah I don't think we need to support a direct upgrade from Airtime to LibreTime with the same exact OS. I mean there is nothing to prevent someone from upgrading their Debian or Ubuntu box to a supported version and then running the Libretime install and upgrading the database but it would be very wise to back the database up at least before doing that.

The media-monitor and watched files not working when imported into LibreTime is probably the biggest potential challenge for someone attempting an upgrade (if they utilized this feature extensively).

**Pull request merged**